include/zephyr/dt-bindings/reset/mchp_mss_reset.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2025 embedded brains GmbH & Co. KG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_RESET_MCHP_MSS_RESET_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_RESET_MCHP_MSS_RESET_H_
+
+/*
+ * The reset ID encodes the bit index of the SUBBLK_CLOCK_CR and SOFT_RESET_CR
+ * registers associated with the device.
+ */
+#define MSS_RESET_ID_ENVM    0x0
+#define MSS_RESET_ID_MAC0    0x1
+#define MSS_RESET_ID_MAC1    0x2
+#define MSS_RESET_ID_MMC     0x3
+#define MSS_RESET_ID_TIMER   0x4
+#define MSS_RESET_ID_MMUART0 0x5
+#define MSS_RESET_ID_MMUART1 0x6
+#define MSS_RESET_ID_MMUART2 0x7
+#define MSS_RESET_ID_MMUART3 0x8
+#define MSS_RESET_ID_MMUART4 0x9
+#define MSS_RESET_ID_SPI0    0xa
+#define MSS_RESET_ID_SPI1    0xb
+#define MSS_RESET_ID_I2C0    0xc
+#define MSS_RESET_ID_I2C1    0xd
+#define MSS_RESET_ID_CAN0    0xe
+#define MSS_RESET_ID_CAN1    0xf
+#define MSS_RESET_ID_USB     0x10
+#define MSS_RESET_ID_RSVD    0x11
```

Comment:
This is a reset line which is to the Fabric, we should not reset this line when running so we prob should have a check for it

Reply:
Is there absolutely no use case for clearing this bit? I am not in favour of adding extra code to enforce a policy here. People working with the reset configuration should know what they are doing.

Reply:
While I agree with you that when working with resets, people need to know what they are doing, the Fabric reset is a special case, also I left a point about bit 17 in the clock register being read only. The driver in [linux](https://github.com/linux4microchip/linux/blob/518e58b21b151e23c368a52f90941bd5abfcb23c/drivers/reset/reset-mpfs.c#L145) has the same check